### PR TITLE
Add extra chips to BuildDetails

### LINKED ISSUE #326: Add a link to branch on a page with build
Example: https://cirrus-ci.com/build/5333512213495808

Current appearance:

image

What I'd like to see:

image

### PR DESCRIPTION
**Pull request by RDIL**
Fixes #326 

Signed-off-by: Reece Dunham <me@rdil.rocks>

### CODE REVIEW

File: src/components/builds/BuildDetails.tsx [hunk 1]
```diff
@@ -168,6 +170,10 @@ class BuildDetails extends React.Component<Props> {
           <CardContent>
             <div className={classes.wrapper}>
               <RepositoryNameChip className={classes.chip} repository={build.repository} />
+              <BuildBranchNameChip className={classes.chip} build={build} />
```
Comment (fkorotkov) [inline src/components/builds/BuildDetails.tsx:173]: This information is available under the `changeMessageTitle`. Will it be enough?

Comment (RDIL) [reply src/components/builds/BuildDetails.tsx:173]: Do you mean from a code perspective or visual perspective?

Comment (fkorotkov) [reply src/components/builds/BuildDetails.tsx:173]: Both. Also from UX perspective links below open external links of the brnach/PR and the commit in GH interface where chips always open a Cirrus page.

Comment (RDIL) [reply src/components/builds/BuildDetails.tsx:173]: @AlexWayfer wanted this, plus isn't it just better to have these chips for consistency?

Comment (RDIL) [reply src/components/builds/BuildDetails.tsx:173]: After taking some time to rethink this, I do still believe this change is for the better.  Why not have Cirrus links there, along with GitHub ones?

Comment (fkorotkov) [reply src/components/builds/BuildDetails.tsx:173]: `BuildChangeChip` will lead to the same page. We can add just the `BuildBranchNameChip`. What do you think?

Comment (RDIL) [reply src/components/builds/BuildDetails.tsx:173]: That works fine for me.  I'll change it.

File: src/components/builds/BuildDetails.tsx [hunk 2]
```diff
@@ -168,6 +169,9 @@ class BuildDetails extends React.Component<Props> {
...
+            </div>
+            <div className={classes.wrapper}>
```
Comment (fkorotkov) [inline src/components/builds/BuildDetails.tsx:174]: Is there a need for an additional wrapper?

Comment (RDIL) [reply src/components/builds/BuildDetails.tsx:174]: Design-wise, the task page has 2 rows of chips, this sorts them like that.

Comment (fkorotkov) [reply src/components/builds/BuildDetails.tsx:174]: What do you think of keeping all the chips in a single row? Task page has 6 chips and they just can't fit in one row.

Comment (RDIL) [reply src/components/builds/BuildDetails.tsx:174]: Right, but they are in 2 rows there, and users prefer consistant design from what I've seen.

**Pull request merged**